packages/material-components-web/README.md

```diff
@@ -40,10 +40,12 @@ For example, say you want to use an [icon toggle](../mdc-icon-toggle). Simply re
 DOM, and attach the `data-mdc-auto-init="MDCIconToggle"` attribute.
 
 ```html
-<i class="mdc-icon-toggle material-icons" role="button" aria-pressed="false"
-   aria-label="Add to favorites" tabindex="0"
-   data-toggle-on='{"label": "Remove from favorites", "content": "favorite"}'
-   data-toggle-off='{"label": "Add to favorites", "content": "favorite_border"}'
+<button class="mdc-icon-toggle material-icons" 
```

Comment:
This still says mdc-icon-toggle; I think it should say mdc-icon-button?

Also, remove trailing space after favorite_border